controllers/reviews.js

```diff
@@ -0,0 +1,108 @@
+var Review = require('../models/review');
+var accepts = require('accepts');
+
+/*
+* GET ALL reviews
+* @method GET
+*/
+exports.index = function (req, res, next) {
+    var accept = accepts(req);
+    Review.find(function(err, reviews){
+        if(accept.type(['json', 'html']) == 'json') {
+            res.setHeader('Content-Type', 'application/json')
+            res.status(200).json(reviews);
+        }else {
+            res.setHeader('Content-Type', 'text/html')
+            res.render('reviews/index', {reviews: reviews})
+        }
+    });
+}
+
+/*
+* CREATE a new reviews
+* @method POST
+* @params id
+*/
+exports.create = function (req, res, next) {
+    var review = new Review(req.body);
+    review.save(function(err){
+        if (err) {
+            res.status(400).json('bad review')
```

Comment:
Si jamais tu voudrais renvoyer un 400, il faudrait alors regarder l'erreur et vérifier qu'il s'agit d'un champ manquant dans le schema Mongo que t'as défini. En ce cas, err aura un objet comme ceci :
            
            { [ValidationError: Validation failed]
                  message: 'Validation failed',
                  name: 'ValidationError',
                  errors: 
                   { placeType: 
                      { [ValidatorError: Path `placeType` is required.]
                        message: 'Path `placeType` is required.',
                        name: 'ValidatorError',
                        path: 'placeType',
                        type: 'required',
                        value: undefined } } }

Mais comme je t'ai dit plus en haut, normalement on fait pas un requête pour après vérifier si les paramètres sont OK. Ça fait consommer de la bande passante et un aller retour en base, sachant qu'on peut vérifier ça avant.